stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1591,6 +1594,7 @@ public class ChatClient internal constructor(
             return this
         }
 
+        @Deprecated("Use customizeOkHttpClient to set the timeouts")
```

Comment:
I think we should also deprecate `cdnTimeout` and the corresponding properties in `ChatClientConfig`